nbmolviz/drivers3d.py

```diff
@@ -71,30 +71,40 @@ def set_background_color(self, color, opacity=1.0):
         self.background_color = color
         self.background_opacity = opacity
 
+    def set_clipping(self, near, far):
+        self.viewer('setSlab', [float(near), float(far)])
+
     def set_color(self, color, atoms):
+        self.styles = self.get_styles_for_color(
+                color, atoms, self.styles
+        )
+
+    # Returns new styles after updating the given atoms with the given color
+    def get_styles_for_color(self, color, atoms, styles):
```

Comment:
Yeah, this is fine; it's a actually a `staticmethod` (in python, a `classmethod` is one that gets the class as the first argument). So you can write this as

```python
@staticmethod
def get_styles_for_color(color, atoms, styles):
 ```